src/detect-engine.c

```diff
@@ -2915,55 +2915,53 @@ static int DetectEngineCtxLoadConf(DetectEngineCtx *de_ctx)
         }
     }
 
-    /* parse port grouping whitelisting settings */
+    /* parse port grouping priority settings */
 
     const char *ports = NULL;
-    (void)ConfGet("detect.grouping.tcp-whitelist", &ports);
```

Comment:
we need to handle old configs as well, which means we first read the new `tcp-priorityports` option, but if it doesn't exist, we try `tcp-whitelist`

Btw `tcp-priority-ports` looks better to me.

Reply:
> we need to handle old configs as well, which means we first read the new `tcp-priorityports` option, but if it doesn't exist, we try `tcp-whitelist`
> 

Indeed. Should be more mindful of backwards compatibility. Thanks a lot! 🙇🏽‍♀️ 

> Btw `tcp-priority-ports` looks better to me.

To me as well. Coming up in the next PR.